core/src/main/java/org/kiwix/kiwixmobile/core/history/HistoryCategoryItemViewHolder.kt

```diff
@@ -0,0 +1,23 @@
+package org.kiwix.kiwixmobile.core.history
+
+import android.view.View
+import kotlinx.android.synthetic.main.header_date.header_date
+import org.kiwix.kiwixmobile.core.R
+import org.kiwix.kiwixmobile.core.base.adapter.BaseViewHolder
+import org.threeten.bp.format.DateTimeFormatter
+import org.threeten.bp.LocalDate
+
+class HistoryCategoryItemViewHolder(itemView: View) : BaseViewHolder<String>(itemView) {
+  override fun bind(item: String) {
+    val todaysDate = LocalDate.now()
+    val yesterdayDate = LocalDate.now().minusDays(1)
```

Comment:
`todaysDate.minusDays(1)` works a little bit better as we don't create `now` again